contracts/Lens/VenusLens.sol

```diff
@@ -43,6 +43,75 @@ contract VenusLens is ExponentialNoError {
         uint dailyBorrowXvs;
     }
 
+    struct VTokenBalances {
+        address vToken;
+        uint balanceOf;
+        uint borrowBalanceCurrent;
+        uint balanceOfUnderlying;
+        uint tokenBalance;
+        uint tokenAllowance;
+    }
+
+    struct VTokenUnderlyingPrice {
+        address vToken;
+        uint underlyingPrice;
+    }
+
+    struct AccountLimits {
+        VToken[] markets;
+        uint liquidity;
+        uint shortfall;
+    }
+
+        struct GovReceipt {
```

Comment:
```suggestion
    struct GovReceipt {
```

Reply:
This is strange 🤔 Why didn't prettier pick this up?